src/wallet/rpcwallet.cpp

```diff
@@ -2972,22 +2977,24 @@ void FundTransaction(CWallet* const pwallet, CMutableTransaction& tx, CAmount& f
             coinControl.m_add_inputs = options["add_inputs"].get_bool();
         }
 
-        if (options.exists("changeAddress")) {
-            CTxDestination dest = DecodeDestination(options["changeAddress"].get_str());
+        if (options.exists("changeAddress") || options.exists("change_address")) {
+            const std::string change_address_str = (options.exists("change_address") ? options["change_address"] : options["changeAddress"]).get_str();
```

Comment:
We have at least one of these already; these may deserve a helper function.
```C++
UniValue GetAliasedOption(const UniValue& options, const std::string& name, const std::string& alias)
{
    switch (options.exists(name) + options.exists(alias)) {
    case 0: return NullUniValue;
    case 1: return options.exists(name) ? options[name] : options[alias];
    }
    throw JSONRPCError(RPC_INVALID_PARAMETER, "Cannot use both " + name + " and " + alias + " simultaneously.");
}
```
then this becomes
```C++
UniValue addr = GetAliasedOption(options, "change_address", "changeAddress"));
if (!addr.isNull()) {
    CTxDestination dest = DecodeDestination(addr.get_str());
    ...
```